transfers/transfer.py

```diff
@@ -11,12 +11,15 @@
 import base64
 import logging
 import logging.config  # Has to be imported separately
+import models
```

Comment:
Hm. I've been running `py.test` from the same directory with a python 2.7 or a python 3.5 virtualenv.  For me, if I change it to `import models` I get the error ` ImportError: No module named 'models'` in py3 and it works in py2.  Does it change anything if you move it below the sys.path modification?